vnext/src/Libraries/Components/Picker/PickerWindows.tsx

```diff
@@ -68,7 +72,7 @@ export class Picker extends React.Component<IPickerProps, State> {
         items.push({
           value: child.props.value,
           label: child.props.label,
-          textColor: processColor(child.props.color),
+          textColor: processColor(child.props.color) || undefined,
```

Comment:
This seems like it might be an issue with a Flow maybe type not being ported correctly (and missing `null`)?

Reply:
Oh, like the TS type for `ProcessedColorValue` should include null? That would make sense.

Reply:
I just realized this line will also potentially convert black to undefined, since we're working on numbers... Will change that

Reply:
Changed the interface to allow null instead. Can't remember how Folly chooses to serialize objects with undefined values (convert to null vs omit), but we ensure the color is non-null on native before we do anything with it it looks like.

Reply:
:+1: